src/audio/buffers/comp_buffer.c

```diff
@@ -152,6 +152,19 @@ static int comp_buffer_sink_set_alignment_constants(struct sof_sink *sink,
 	return 0;
 }
 
+static void comp_buffer_clean(struct sof_audio_buffer *audio_buffer)
+{
+	if (!audio_buffer)
```

Comment:
Can you add an inline comment stating why/where its possible to call this API with NULL buffer ?